app/controllers/resource_languages_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ResourceLanguagesController < SecureController
+  before_action :get_resource, :get_language, :build_resource_language
+
+  def show
+    render json: @resource_language, include: params[:include], status: :ok
+  end
+
+  def update
+    @resource_language.set_data_attributes!(data_attrs)
+    render json: @resource_language, status: :ok
+  end
```

Comment:
when I send this request:
```
PUT /resources/:resource_id/languages/:language_id
Content-Type: application/vnd.api+json

{
    "data": {
        "type": "resource-language",
        "attributes": {
            "attr-test": true
        }
    }
}
```

it returns this attribute: `"attr-test": "t"`, I'm guessing that is an artifact of how ruby represents true/false. can that scenario be transformed into the string "true" instead of "t"

Reply:
In addition when I send this request:
```
PUT /resources/:resource_id/languages/:language_id
Content-Type: application/vnd.api+json

{
    "data": {
        "type": "resource-language",
        "attributes": {
            "attr-test": false
        }
    }
}
```
it deletes `attr-test` instead of setting the value to "false"

Reply:
Good catch, I've fixed this and added a test for it